### ISSUE TITLE
Error occurs after closing CherryTree

### ISSUE THREAD

**Issue by whalehub** [giuspen/cherrytree#435]
I'm running Windows 10 Pro on my computer with CherryTree v0.38.6. Whenever I close CherryTree, I receive the following error message:

![cherrytree_2018-12-01_17-21-28](https://user-images.githubusercontent.com/44198148/49330351-8b3c9f80-f58d-11e8-8959-dfa31db991f5.png)

This problem affects both the portable and the installer version of this software. Here is the output from the logfile that is mentioned in the error message:

```
Traceback (most recent call last):
  File "core.pyc", line 3026, in on_node_changed
  File "core.pyc", line 4977, in update_selected_node_statusbar_info
  File "support.pyc", line 44, in get_timestamp_str
TypeError: decode() argument 1 must be string, not None
```

**Comment by giuspen** [comment]
This should be already fixed, I have to release the new version shortly

**Comment by whalehub** [comment]
This issue is fixed with v0.38.7.
